config/install/block.block.localgov_services_block.yml

```diff
@@ -0,0 +1,28 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - views.view.services
+  module:
+    - system
+    - views
+    - localgov_core
+  theme:
+    - localgov_theme
+id: views_block__services_block_service_list
+theme: localgov_theme
+region: content_top
+weight: -4
```

Comment:
The Service list block is appearing **above** the Welcome message block :(  Can this weight be give a positive number please.